arch/riscv/Kconfig

```diff
@@ -238,6 +238,15 @@ config RISCV_HART_MASK
 	  i.e. 128, 129, ..(0x80, 8x81, ..), this can be configured to 63 (0x7f)
 	  such that we can extract the bits that start from 0.
 
+config EXTRA_EXCEPTION_INFO
+	bool "Collect extra exception info"
+	default y if !RISCV_SOC_HAS_ISR_STACKING
```

Comment:
SoCs with `RISCV_SOC_HAS_ISR_STACKING` defines their own `struct arch_esf` in `soc/`, so do not enable `EXTRA_EXCEPTION_INFO` by default if `RISCV_SOC_HAS_ISR_STACKING` is set.

These SoC can select/enable `EXTRA_EXCEPTION_INFO` manually if `csf` exists in their `arch_esf` declaration.

Reply:
Removed the `default y`, extra stuff probably shouldn't be enabled by default